server/models/user.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+const {
+  Model
+} = require('sequelize');
+const {makeHash} = require('../helpers/bcrypt')
+module.exports = (sequelize, DataTypes) => {
+  class User extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      // define association here
+    }
+  };
+  User.init({
+    email: {
+      type: DataTypes.STRING,
+      validate: {
+        isEmail: {
+          args: true,
+          msg: 'invalid email format'
+        }
+      },
+      unique: true
+    },
+    password: DataTypes.STRING
```

Comment:
ni password gua isi 1 gapapa ya ?

Reply:
untuk password sudah dikasih not empty dan minimal karakter kak